### ISSUE TITLE
Training loss not converging

### ISSUE THREAD

**Issue by SatanuG** [facebookresearch/crystal-text-llm#14]
Hi,

I tried to replicate your experiments, but unfortunately, I could not get the same loss curves you reported in the paper for the Llama-13b model. Here is my training log: [https://api.wandb.ai/links/satanu-ghosh-university-of-new-hampshire/uc1xrqdc](url).

It would be of great help to me if you could answer two questions:

1. Which model loss curves did you use: the regular or the chat model?
2. Did you do any gradient clipping (besides the default value set in the Huggingface trainer module of 1)?

Thanks for your help!

--Satanu.
